tools/travis/build_byond.sh

```diff
@@ -5,29 +5,13 @@ set -e
 shopt -s globstar
 
 if [ "$BUILD_TOOLS" = false ]; then
-	if grep '"aaa" = \(.+\)' _maps/**/*.dmm;	then
```

Comment:
Keep this one in.

Reply:
TGM is already checked on the script though.

Reply:
Oh wait this hooks into travis, nvm